# Bioinformatics

# Table of content

* [Repositories](#repositories)
* [Videos](#videos)
* [Misc](#misc)

## Repositories

- `ads1-slides` - Ben Langmead's Slides for Algorithms for DNA Sequencing Coursera class. [https://github.com/BenLangmead/ads1-slides.git](https://github.com/BenLangmead/ads1-slides.git). The complete class is at [http://www.langmead-lab.org/teaching-materials/](http://www.langmead-lab.org/teaching-materials/)

- `alignment-and-variant-calling-tutorial` - basic walk-throughs for alignment and variant calling from NGS sequencing data, by Erik Garrison. [https://github.com/ekg/alignment-and-variant-calling-tutorial](https://github.com/ekg/alignment-and-variant-calling-tutorial)

- `angus` - Materials for Analyzing Next-Generation Sequencing (ANGUS) course. [https://github.com/ngs-docs/angus](https://github.com/ngs-docs/angus)

- `applied-computational-genomics` - applied computational genomics by Aaron Quinlan, https://github.com/quinlan-lab/applied-computational-genomics

- `appliedgenomics` - JHU Applied Comparative Genomics 2017 course, by Michael Schatz. https://github.com/schatzlab/appliedgenomics, links to similar courses there. Other courses by Michael, http://schatzlab.cshl.edu/teaching/

- `awesome_genome_browsers` - genome browsers and genomic visualization tools. https://github.com/davemcg/awesome_genome_browsers

- `awesome-10x-genomics` - List of tools and resources related to the 10x genomics GEMCode/Chromium system. https://github.com/johandahlberg/awesome-10x-genomics

- `awesome-cancer-variant-databases` - A community-maintained repository of cancer clinical knowledge bases and databases focused on cancer variants. https://github.com/seandavi/awesome-cancer-variant-databases

- `awesome-expression-browser` - A curated list of software and resources for exploring and visualizing (browsing) expression data, and more. https://github.com/federicomarini/awesome-expression-browser

- `awesome-deepbio` - A curated list of awesome deep learning applications in the field of computational biology. https://github.com/gokceneraslan/awesome-deepbio

- `awesome-microbes` - List of software packages (and the people developing these methods) for microbiome (16S), metagenomics (WGS, Shot-gun sequencing), and pathogen identification/detection/characterization. [https://github.com/stevetsa/awesome-microbes](https://github.com/stevetsa/awesome-microbes)

- `awesome-reproducible-research` - A curated list of reproducible research case studies, projects, tutorials, and media. [https://github.com/leipzig/awesome-reproducible-research](https://github.com/leipzig/awesome-reproducible-research)

- `awesome-single-cell` - List of software packages for single-cell data analysis, including RNA-seq, ATAC-seq, etc. [https://github.com/seandavi/awesome-single-cell](https://github.com/seandavi/awesome-single-cell)

- `atac_dnase_pipelines` - ATAC-seq and DNase-seq processing pipeline by Anshul Kundaje, [https://github.com/kundajelab/atac_dnase_pipelines](https://github.com/kundajelab/atac_dnase_pipelines)

- `ATAC_Workshop` - ATAC_seq Workshop. [https://github.com/ThomasCarroll/ATAC_Workshop](https://github.com/ThomasCarroll/ATAC_Workshop)

- `bcbio-nextgen` - Validated, scalable, community developed variant calling and RNA-seq analysis https://bcbio-nextgen.readthedocs.org [https://github.com/chapmanb/bcbio-nextgen](https://github.com/chapmanb/bcbio-nextgen)

- `bio_610B` - Course materials for Advanced Biological Statistics II by Peter Ralph. Using STAN. https://github.com/petrelharp/bio_610B 

- `BiocWorkshops` - 2018 Bioconductor Workshops. The book by Martin Morgan: https://bioconductor.github.io/BiocWorkshops/, GitHub: https://github.com/Bioconductor/BiocWorkshops

- `bioinformatics-one-liners` - collection of bioinformatics-genomics bash one liners, using awk, sed etc. [https://github.com/crazyhottommy/bioinformatics-one-liners](https://github.com/crazyhottommy/bioinformatics-one-liners)

- `Bioinformatics-Resources` -  A curated list of resources for learning bioinformatics. [https://github.com/YTLogos/Bioinformatics-Resources](https://github.com/YTLogos/Bioinformatics-Resources)

- `biostar-handbook-web` - Biostar Handbook - bioinformatics survival guide. A practical overview for the data analysis methods of bioinformatics. [https://www.biostarhandbook.com/index.html](https://www.biostarhandbook.com/index.html) and on Git [https://github.com/ialbert/biostar-handbook-web](https://github.com/ialbert/biostar-handbook-web)

- `biotools` - A massive collection of references on the topics of bioinformatics, sequencing technologies, programming, machine learning, and more. [https://github.com/jdidion/biotools](https://github.com/jdidion/biotools)

- `book` - "Computational genomics with R" book by Altuna Akalin. Web site, [https://compgenomr.github.io/book/](https://compgenomr.github.io/book/), and GitHub repo, [https://github.com/compgenomr/book](https://github.com/compgenomr/book)

- `ChIP-seq-analysis` - ChIP-seq analysis notes from Tommy Tang, [https://github.com/crazyhottommy/ChIP-seq-analysis](https://github.com/crazyhottommy/ChIP-seq-analysis)

- `comp-genomics-class` - Computational genomics, algorithms, by Ben Langmead. In Python. [https://github.com/BenLangmead/comp-genomics-class](https://github.com/BenLangmead/comp-genomics-class)

- `compbio_src` - Introduction to Computational Biology - source Rmarkdown files. https://github.com/biodatascience/compbio_src, web-site, https://biodatascience.github.io/compbio/

- `statcomp_src` - Statistical Computing - source Rmarkdown files. https://github.com/biodatascience/statcomp_src, web-site, https://biodatascience.github.io/statcomp/

- `CSE549` - CSE 549 - Introduction to Computational Biology by Steven Skiena. Includes video lectures. [http://www3.cs.stonybrook.edu/~skiena/549/](http://www3.cs.stonybrook.edu/~skiena/549/)

- `deeplearning-biology` - A list of deep learning implementations in biology [https://github.com/hussius/deeplearning-biology](https://github.com/hussius/deeplearning-biology)

- `DIYtranscriptomics.github.io` - Course material for the "do-it-yourself" RNA-seq course, by Daniel Beiting. https://github.com/DIYtranscriptomics/DIYtranscriptomics.github.io, http://diytranscriptomics.com/

- `DNA-seq-analysis` - Notes on whole exome and whole genome sequencing analysis. [https://github.com/crazyhottommy/DNA-seq-analysis](https://github.com/crazyhottommy/DNA-seq-analysis)

- `DS_for_S` - Materials for March 12 ENAR short course on Data Science for Statisticians. [https://github.com/AmeliaMN/DS_for_S](https://github.com/AmeliaMN/DS_for_S)

- `genbioconductor` - "Materials for Genomics Data Science: Introduction to Bioconductor" course by Kasper Hansen. Includes videos, code examples and lecture material. Web-site, [https://kasperdanielhansen.github.io/genbioconductor/](https://kasperdanielhansen.github.io/genbioconductor/), and GitHub repo, [https://github.com/kasperdanielhansen/genbioconductor](https://github.com/kasperdanielhansen/genbioconductor)

- `GEN242` - Thomas Girke' "Class: Data Analysis in Genome Biology (GEN242)" - bioinformatics of NGS data analysis [https://github.com/tgirke/GEN242](https://github.com/tgirke/GEN242)

- `genomicspapers` - Jeff Leek recommended list of genomics papers. [https://github.com/jtleek/genomicspapers](https://github.com/jtleek/genomicspapers)

- `getting-started-with-genomics-tools-and-resources` - links and references to many resources [https://github.com/crazyhottommy/getting-started-with-genomics-tools-and-resources](https://github.com/crazyhottommy/getting-started-with-genomics-tools-and-resources)

- `GWAStutorial` - code for GWAS tutorial blog post [https://poissonisfish.wordpress.com/2017/10/09/genome-wide-association-studies-in-r/](https://poissonisfish.wordpress.com/2017/10/09/genome-wide-association-studies-in-r/), [https://github.com/monogenea/GWAStutorial](https://github.com/monogenea/GWAStutorial)

- `haowulab.github.io` - microarray course, https://github.com/haowulab/haowulab.github.io

- `INF-BIOx121_fall2014_de_novo_assembly` - _de novo_ assembly presentations and Jupyter notebooks tutorials. https://github.com/lexnederbragt/INF-BIOx121_fall2014_de_novo_assembly. The full "High Throughput Sequencing technologies and bioinformatics analysis" course material is at https://wiki.uio.no/projects/clsi/index.php/INF-BIOX121_H14. The latest version is at https://github.com/lexnederbragt/INF-BIOx121, in the branch corresponding to year. Course web-site, https://inf-biox121.readthedocs.io/en/2017/

- `In-depth-NGS-Data-Analysis-Course` - The course is aimed at bench biologists who are interested in learning about NGS-based genomic analysis. The topics covered in-depth during this course are analysis of RNA-Seq and ChIP-Seq data, with an optional Variant Calling session. https://github.com/hbctraining/In-depth-NGS-Data-Analysis-Course

- `jhudatascience` - links to several courses, including some material. JHU. The GitHub repo [https://github.com/jhudsl/jhudatascience](https://github.com/jhudsl/jhudatascience) contains the files to build the web-site [http://jhudatascience.org/courses.html](http://jhudatascience.org/courses.html)

- `Machiraju` - Courses by Dr. Raghu Machiraju, http://web.cse.ohio-state.edu/~machiraju.1/. Topics: data visualization, biomedical informatics, computer graphics, linked from the homepage. These and other courses are linked from http://web.cse.ohio-state.edu/~machiraju.1/teaching/. 
  - `CSE5599-BMI7830` - biomedical informatics. http://web.cse.ohio-state.edu/~machiraju.1/teaching/CSE5599-BMI7830/
  - `CSE5544` – Introduction to Data Visualization. http://web.cse.ohio-state.edu/~machiraju.1/teaching/CSE5544/

- `labs` - Biomedical Data Science PH525x course from Rafael Irizarry and Michael Love [https://genomicsclass.github.io/book/](https://genomicsclass.github.io/book/) and [https://github.com/genomicsclass/labs](https://github.com/genomicsclass/labs). Videos http://lorainelab.org/statistics-and-r-for-the-life-sciences/. Data science book `dsbook`, [https://github.com/rafalab/dsbook](https://github.com/rafalab/dsbook)

- `LinearModelingR` - Linear Modeling in R workshop materials, https://github.com/stephenturner/LinearModelingR

- `msu_ngs20151` - RNA-seq differential expression analysis, https://github.com/mistrm82/msu_ngs2015

- `networks-across-scales` - CMSC423: Bioinformatics algorithms, databases and tools (Spring 2017) by Hector Corrada, https://github.com/hcorrada/networks-across-scales

- `R_programs` - PDFs and statistical learning implementations, data mining, and other statistical analysis programmed in R, https://github.com/a-rhodes-vcu/R_programs

- `RNA-seq-analysis` - RNAseq analysis notes from Ming Tang, [https://github.com/crazyhottommy/RNA-seq-analysis](https://github.com/crazyhottommy/RNA-seq-analysis)

- `rnaseq_tutorial` - Informatics for RNA-seq, by Griffith lab, [https://github.com/griffithlab/rnaseq_tutorial](https://github.com/griffithlab/rnaseq_tutorial)

- `rnaseq_tutorial.wiki` - wiki has the detailed tutorials. "LectureFiles" have lecture slides from different courses

- `SBG-CGC_course2018` - IARC course on analyzing TCGA data in the SevenBridges Genomics Cancer Genomics Cloud (SBG-CGC). [https://github.com/IARCbioinfo/SBG-CGC_course2018](https://github.com/IARCbioinfo/SBG-CGC_course2018)

- `scATAC-seq-analysis-notes` - single-cell ATAC-seq notes by Ming Tang, https://github.com/crazyhottommy/scATAC-seq-analysis-notes

- `scRNA.seq.course` - Analysis of single cell RNA-seq data course, Cambridge University, UK. [https://github.com/hemberg-lab/scRNA.seq.course](https://github.com/hemberg-lab/scRNA.seq.course) and its bookdown version [https://hemberg-lab.github.io/scRNA.seq.course/](https://hemberg-lab.github.io/scRNA.seq.course/)

- `scTeachingMaterials` - Teaching materials for single-cell RNA-seq data analysis courses. Includes lectures and practicals. Data download scripts are provided. [https://github.com/MarioniLab/scTeachingMaterials](https://github.com/MarioniLab/scTeachingMaterials)

- `single_cell_analysis` - Workshops on single cell analysis. Presentation, R code with instructions for running a Docker container on Amazon EC2. https://github.com/broadinstitute/single_cell_analysis

- `single-cell-pseudotime` - Single-cell RNA-seq pseudotime estimation algorithms, comprehensive collection of links to software and accompanying papers. https://github.com/agitter/single-cell-pseudotime

- `Statistical_Genetics_Book` - Statistical genetics book, by [Shirin Glander](https://github.com/ShirinG). [https://github.com/ShirinG/Statistical_Genetics_Book](https://github.com/ShirinG/Statistical_Genetics_Book)

- `stats337` - Readings in Applied Data Science, collection of papers assembled by Hadley Wickham, [https://github.com/hadley/stats337](https://github.com/hadley/stats337)

- `workshops` - Material for the THRIV Biomedical Data Science 2018 course by Stephen Turner [https://github.com/stephenturner/workshops](https://github.com/stephenturner/workshops), web site [https://thriv.github.io/biodatasci2018/](https://thriv.github.io/biodatasci2018/)


- `ChIP-seq_workflow.pdf` - ChIP-seq workflow and analysis in R, https://www.bioconductor.org/help/course-materials/2017/OMRF/B4_ChIPSeq.html

- `RNA-seq-introduction-v2.pdf` - 33-pages RNA-seq introduction. [https://www.princeton.edu/genomics/sequencing/instructions/rna-seq-workshop/RNA-seq-introduction-v2.pdf](https://www.princeton.edu/genomics/sequencing/instructions/rna-seq-workshop/RNA-seq-introduction-v2.pdf)

- `RNA-seq_diff_expr.Rmd` - RNA-seq: differential gene expression analysis, https://gist.github.com/stephenturner/e34e32b3d054bb850ae2

- `RNA-seq_Bioconductor.pdf` - Bioconductor building blocks for RNA-seq data analysis, https://www.bioconductor.org/help/course-materials/2017/OMRF/B2_Common_Operations.html

- `RNA-seq_workflow.pdf` - SummarizedExperiment and RNA-seq data manipulation in Bioconductor, https://www.bioconductor.org/help/course-materials/2017/OMRF/B3_RNASeq_Workflow.html

## Videos

- `Bioconductor Workshop 2: RNA Seq and ChIP Seq Analysis` - 6 hours workshop on RNA-seq and ChIP-seq technology and analysis by Levi Waldron and others. https://www.youtube.com/watch?v=J9LNYhhNhrk

- “Differential Splicing Analysis with RNA-Seq: Current Applications, Approaches, & Limitations” - 1 hour overview of differential splicing analysis. https://www.youtube.com/watch?v=LknDQw08P5w

- `NHGRI_Genomics2016` - "Current Topics in Genome Analysis 2016" course. A lecture series covering contemporary areas in genomics and bioinformatics. Video lectures, https://www.youtube.com/playlist?list=PL1ay9ko4A8skYqjhrA4INDZ7IHtebS0lY, and slides, https://www.genome.gov/12514288/current-topics-in-genome-analysis-2016-course-syllabus-handouts-and-videos/

- `MIT_SysBiol2014` - "Foundations of Computational and Systems Biology". Video lectures, https://www.youtube.com/watch?v=lJzybEXmIj0&list=PLUl4u3cNGP63uK-oWiLgO7LLJV6ZCWXac, and slides https://ocw.mit.edu/courses/biology/7-91j-foundations-of-computational-and-systems-biology-spring-2014/lecture-slides/

- "Regulatory Genomics and Epigenomics" - series of genomcs-oriented talks by Simons Institute. 26 videos, ~30min each. https://www.youtube.com/playlist?list=PLgKuh-lKre10fYXnD-8ghi9b9xl83CKet

- "Foundations of Data Science — Spring 2016" course from UC Berkeley. Instructor: John DeNero. Video and slides. [https://data-8.appspot.com/sp16/course](https://data-8.appspot.com/sp16/course). This course is accompanied by other "connector" courses from UC Berkeley, [https://data-8.appspot.com/sp16/modules/extra_tabs/render?index=3](https://data-8.appspot.com/sp16/modules/extra_tabs/render?index=3)

- "[A Roadmap to the Living Genome](https://videocast.nih.gov/summary.asp?Live=12792&bhcp=1)" by [John Stamatoyannopoulos](http://www.stamlab.org/). An overview of cell type-specific (epi)genomic landscapes, visualization and analysis techniques, association and enrichment of disease-associated variants in regulatory regions.

- "RNA-Seq Methods and Algorithms" short video course by Harold Pimentel, pseudoalignment, kallisto, sleuth, practical. [https://www.youtube.com/watch?v=96yBPM8lEt8&list=PLfFNmoa-yUIb5cYG2R1zf5rtrQQKZvKwG](https://www.youtube.com/watch?v=96yBPM8lEt8&list=PLfFNmoa-yUIb5cYG2R1zf5rtrQQKZvKwG)

## Misc

- Computational Genomics With R, online book by Altuna Akalin. http://compgenomr.github.io/book/, https://github.com/compgenomr/book

- Orchestrating Single-Cell Analysis with Bioconductor, online book by Robert Amezquita and Stephanie Hicks. https://bioconductor.github.io/OrchestratingSingleCellAnalysis/

- List of software packages for multi-omics analysis. https://github.com/mikelove/awesome-multi-omics

- Figure depicting the breadth of multimodal scRNA-seq technologies. https://github.com/arnavm/multimodal-scRNA-seq

- List of existing genome analysis plaforms - https://docs.google.com/spreadsheets/d/1o8iYwYUy0V7IECmu21Und3XALwQihioj23WGv-w0itk/pubhtml

- Beginner's Handbook to Next Generation Sequencing by GenoHub - https://genohub.com/next-generation-sequencing-handbook/

- "RNA-seqlopedia" - one-pager of all steps of RNA-seq data analysis, from molecular biology to computational analysis, very detailed. [http://rnaseq.uoregon.edu/](http://rnaseq.uoregon.edu/)

- Informatics for RNA-seq: A web resource for analysis on the cloud. [https://github.com/griffithlab/rnaseq_tutorial](https://github.com/griffithlab/rnaseq_tutorial)

- "Advanced Data Analysis from an Elementary Point of View" book by Cosma Rohilla Shalizi, PDF, 856 pages [http://www.stat.cmu.edu/~cshalizi/ADAfaEPoV/ADAfaEPoV.pdf](http://www.stat.cmu.edu/~cshalizi/ADAfaEPoV/ADAfaEPoV.pdf)

- `BaRC Hot Topics` - lecture slides and handouts on all genomics topics, from Unix to microarray, sequencing, genomics and statistics, http://barc.wi.mit.edu/education/hot_topics/

- The [Bioinformatics algorithms](http://bioinformaticsalgorithms.com/index.htm) web site. [Videos](http://bioinformaticsalgorithms.com/videos.htm) and [Texts](http://bioinformaticsalgorithms.com/faqs.htm) covering topics from biology to genome assembly algorithms, motif finding, sequence comparisons and more bioinformatics tasks and solutions.